As of Python 2.6, "raise" no longer accepts a literal string; you have
to create an Exception object out of it.  (#585358)


--- a/contrib/server-side/fsfsverify.py
+++ b/contrib/server-side/fsfsverify.py
@@ -978,7 +978,7 @@ def fixStream(e, revFile):
       break
 
   if not m:
-    raise "Couldn't find end of rep!"
+    raise Exception("Couldn't find end of rep!")
 
   finalOffset = errorOffset + srcLength
   srcOffset = errorOffset
--- a/contrib/server-side/svn-fast-backup
+++ b/contrib/server-side/svn-fast-backup
@@ -257,7 +257,7 @@ def do_rsync_backup():
 
     # copy db/current.  -R tells rsync to use relative pathnames.
     if runcmd([rsync, '-aR', 'db/current', rsync_dest]):
-        raise "%s: rsync failed" %sys.argv[0]
+        raise Exception("%s: rsync failed" %sys.argv[0])
 
     # Now copy everything else.
     cmd = [rsync, '-a',
@@ -270,7 +270,7 @@ def do_rsync_backup():
         cmd += ['--link-dest', os.path.join(options.backup_dir, previous_backups[-1])]
 
     if runcmd(cmd):
-        raise "%s: rsync failed" %sys.argv[0]
+        raise Exception("%s: rsync failed" %sys.argv[0])
 
     # Rename to final name.
     if not options.simulate:
